### ISSUE TITLE
conflict with jQuery

### ISSUE THREAD

**Issue by kimas-m** [daleharvey/pacman#8]
when I use your code with some simple (fadeI; fadeOut) jQuery i get "undefined is not a function"  it points to this line "var collection = ( tweeners[ prop ] || [] ).concat( tweeners[ "*" ] )," on jQuery. It works when i dont use any of game code. How to solve this problem because i want to use jQuery code too

**Comment by dinc5150** [comment]
@kimasgold Did you find a way around it, so that you could use jQuery?

**Comment by dinc5150** [comment]
Added pull request to fix this issue #14

**Comment by kimas-m** [comment]
Yes I think i found a way around it, that it wont interfere with jquery. Just now i cant remember this at all. sorry. back then i was frustrating on this matter.  but i can see you found a way to fix it too.

**Comment by dinc5150** [comment]
Thanks @kimasgold . Yes, I a "Clone" method added to the Object.prototype and jQuery was having issues with the function being added to all objects. As the method was only ever used once in the code, I simply changed to to be a standalone method and passed the object to it. All my jQuery animations are now working fine with the update.

**Comment by johannesE** [comment]
Please merge this. Because of this, I fucked up my production environment on April 1st. 🙄
Should have done proper testing..
